spec/order_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'order'
+
+describe Order do
+  let(:menu) do
+    double :menu,
+    menu: [{dish: 'BBQ',       price: 10},
+           {dish: 'Sushi',     price: 6}]
+  end
+
+  describe "#choose_dish" do
+    it "has customer choose a dish and quantities" do
+      expect(subject.choose_dish('BBQ', 3))
+        .to eq([{dish: 'BBQ', quantities: 3}])
+    end
+  end
+
+  it "checks your orders" do
+    subject.choose_dish('BBQ', 4)
+    subject.choose_dish('Sushi', 4)
+    expect(subject.check_orders(Menu.new))
+      .to eq("You will be charged £64.")
```

Comment:
Style/DotPosition: Place the . on the previous line, together with the method call receiver.